CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -pedantic -Wall -Wextra")
```

Comment:
Oops, my bad. I had assumed I copied the flags over verbatim, but I guess I missed one. Will do!